website/views/layouts/layout.ejs

```diff
@@ -123,7 +123,8 @@
 
       <%- body %>
 
-      <div style="background-color: #182147;" purpose="page-footer">
+      <%/* Note: footer is hidden until the page is loaded. See <script> tag at the bottom of this file. */%>
+      <div style="background-color: #182147;" class="invisible" purpose="page-footer" data-hide-until-rendered>
```

Comment:
hmm... actually, check this out:
![image](https://user-images.githubusercontent.com/618009/140851432-6b5bbe6c-7152-4c56-9d66-1bbaf071928a.png)

Maybe one of the quotes is a problem?

If not this, then could be it's getting confused by the EJS comment (though we are using them already, so shouldn't be that.)

Reply:
this was because I was using '<script>' in the comment, it has been updated to be '«script»'

Reply:
💯 nice catch-- this has bitten me before, totally forgot it was a thing